server/src/main/java/io/mantisrx/master/jobcluster/job/IMantisJobMetadata.java

```diff
@@ -37,6 +37,8 @@
  */
 public interface IMantisJobMetadata {
 
+    long NOT_SET = 0;
```

Comment:
this could be more explicit `DEFAULT_STARTED_AT_EPOCH` and `DEFAULT_ENDED_AT_EPOCH`. Also, it seems it was set to `-1` previously

Reply:
will do.